### PR TITLE
Note archival node storage size is an ever-growing number, add header…

### PR DESCRIPTION
**Pull request by clay-aptos**
… for network types

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ On the one hand, network storage requires additional CPU support to scale IOPS,
 
 When you are running a validator node, you are required to open network ports on your node to allow other nodes to connect to you. For fullnodes this is optional.
 
-There are three types of Aptos networks. Your node can be configured so that each of these networks can connect to your node using a different port on your node.
+### Network types
+
+There are three types of Aptos networks. Your node can be configured so that each of these networks can connect to your node using a different port on your node:
```
Comment (msmouse) [inline developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md:60]: > Make sure to keep the TCP port 6186 open for the local backup storage service.

Can we change this line below as well? -- it sounds like we suggest to open that port.

Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md:60]: Thanks, @msmouse . Should I remove it entirely? I see no other reference to it on the site.

Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md:60]: And should I remove it for both node types, @msmouse ?

Comment (msmouse) [reply developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md:60]: Yes let's remove. When I and @areshand  have a better story around opening the backup & restore tooling to the community, we can add it back properly.

Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md:60]: Thanks, @msmouse ! All done.

**Review by saharct — APPROVED** [review]

**Pull request merged**